### PR TITLE
add a roadmap

### PR DESCRIPTION
**Pull request by raphjaph**
Refined and added some stuff to the notes I took the other day.

#540

### CODE REVIEW

**Comment by raphjaph** [comment]
Still have to integrate on website somewhere but just wanted to show you the text if you want to add/delete stuff.

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Ordinals Medium-Term Roadmap
```
Comment (casey) [inline ROADMAP.md:1]:
```suggestion
Ordinals Roadmap
================
```

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Ordinals Roadmap
+================
+
+## 1. MVP
```
Comment (casey) [inline ROADMAP.md:4]:
```suggestion
MVP
---
```

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+- make it easy and private for people to check whether their UTXOs contain cool/
+rare ordinals
+- send and receive ordinals
+  - finish wallet
+  - write how to do with createrawtransaction?
+
+
+## 2. Gain user mindshare
```
Comment (casey) [inline ROADMAP.md:14]:
```suggestion
Shill
-----
```

File: ROADMAP.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+Shill
+-----
+
+- memes
+- twitter bot
+- blog about ordinals
+- create a market for ordinals -> bounties
+
+
+## 3. Increase Dev Participation
```
Comment (casey) [inline ROADMAP.md:23]:
```suggestion
Onboard Developers
------------------
```

File: ROADMAP.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+Ordinals Roadmap
+================
+
+MVP
+---
```
Comment (casey) [inline ROADMAP.md:5]:
```suggestion
Wallet MVP
----------
```

File: ROADMAP.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+Wallet MVP
+----------
+
+- make it easy and private for people to check whether their UTXOs contain cool/
+rare ordinals
+- send and receive ordinals
+  - finish wallet
+  - write how to do with createrawtransaction?
+
```
Comment (casey) [inline ROADMAP.md:12]:
```suggestion
```

File: ROADMAP.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
+Ordinals Roadmap
+================
+
+Wallet MVP
+----------
+
+- make it easy and private for people to check whether their UTXOs contain cool/
+rare ordinals
+- send and receive ordinals
+  - finish wallet
```
Comment (casey) [inline ROADMAP.md:10]:
```suggestion
- send and receive ordinals with bitcoin core-based wallet
```

File: ROADMAP.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+rare ordinals
+- send and receive ordinals with bitcoin core-based wallet
+
+Shill
+-----
+
+- memes
+- twitter bot
+- blog about ordinals
+- create a market for ordinals -> bounties
```
Comment (casey) [inline ROADMAP.md:17]:
```suggestion
- keep doing bounties
- create a market for ordinals
```

File: ROADMAP.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+Shill
+-----
+
+- memes
+- twitter bot
+- blog about ordinals
+- keep doing bounties
+- create a market for ordinals
+
```
Comment (casey) [inline ROADMAP.md:19]:
```suggestion
```

File: ROADMAP.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+Onboard Developers
+------------------
+
+- onboard new contributors
+- "Ordinals Coding Club"/"Deciphering Ordinals"/(Find a cool name):
+  - streaming on Twitter Spaces & Discord
```
Comment (casey) [inline ROADMAP.md:25]:
```suggestion
- keep streaming on Twitter Spaces & Discord
```

File: ROADMAP.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+- create a market for ordinals
+
+Onboard Developers
+------------------
+
+- onboard new contributors
+- keep streaming on Twitter Spaces & Discord
+- make development easy and fast
+  - fast unit tests 
+  -
```
Comment (casey) [inline ROADMAP.md:27]:
```suggestion
- make development easy and fast
```

**Pull request merged**